modules/xmpp/JingleSession.js

```diff
@@ -56,14 +56,21 @@ function JingleSession(me, sid, peerjid, connection,
      * @type {JingleSessionState}
      */
     this.state = null;
+
+    /**
+     * The RTC service instance
+     * @type {RTC|null}
```

Comment:
i think in general the convention has been to only document its intended type, and not denote it could be null (i don't think we do that anywhere else, and i don't think we have any type that can't be null, so maybe the `|null` is redundant?)